src/applications/personalization/rated-disabilities/components/RatedDisabilityView.jsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import RatedDisabilityTable from './RatedDisabilityTable';
+
+class RatedDisabilityView extends React.Component {
+  static propTypes = {
+    fetchRatedDisabilities: PropTypes.func.isRequired,
+    ratedDisabilities: PropTypes.shape({
+      ratedDisabilities: PropTypes.array,
+    }),
+  };
+
+  render() {
+    const { fetchRatedDisabilities, ratedDisabilities, user } = this.props;
+
+    let content;
+
+    // Total Disability Calculation and Pending Disabilities should go here.
+    if (user.profile.verified) {
+      if (user.profile.status === 'OK') {
+        content = (
+          <>
+            <RatedDisabilityTable
+              fetchRatedDisabilities={fetchRatedDisabilities}
+              ratedDisabilities={ratedDisabilities}
+            />
+          </>
+        );
+      }
+    }
+
+    return (
+      <div className="row">
+        <div>{content}</div>
+      </div>
+    );
```

Comment:
I'm guessing this `content` var is just a placeholder for future user states?

Reply:
Currently, `content` actually contains the child component used above on line 21. I think I was just following a similar convention I noticed in `applications/personalization/profile360/components/ProfileView`. I was thinking that `content` may show something else down the line, in the event something is wrong with the user profile.

Reply:
This could be inlined with a ternary- requires less scrolling up and and down esp if more conditionals are added.

```
{/* Total Disability Calculation and Pending Disabilities should go here. /*}
{user?.profile?.verified && 
user?.profile?.status === 'OK' &&
<RatedDisabilityTable
  fetchRatedDisabilities={fetchRatedDisabilities}
  ratedDisabilities={ratedDisabilities}
/>}
```

Not sure what the structure of the props are further upstream but it might be better to flatten these props out and name them closer to the view behavior e.g. `showRatedDsiabilityTable` would be passed in a prop and equal `user.profile.verifed && user.profile.status === 'OK'` (defined in something like a reducer or `mapStateToProps`). Coupling the view to `user` can make it harder to maintain and understand.